src/js/components/Box/Box.js

```diff
@@ -93,14 +90,6 @@ const Box = forwardRef(
       }
     }, [refProp, as, responsiveProp]);
 
-    useEffect(() => {
-      if (skeletonProp?.message?.start) announce(skeletonProp.message.start);
-      else if (typeof skeletonProp?.message === 'string')
```

Comment:
I realized we have a prop on Box for passing messages to skeleton, we'll need to figure out how we want to handle the backwards compatibility of that

Reply:
I'm thinking if someone passes skeleton.message we should go ahead and leave the announcement to maintain backwards compatibility. Also I was thinking about this more and even though the skeleton itself should be hidden from the screenreader user, an announcement indicating something has loaded could be useful in certain situations. The implementor should use it sparingly though so they don't overload the screen reader user with unnecessary announcements

Reply:
done